ecs-agent/api/errors/errors.go

```diff
@@ -58,7 +58,15 @@ func IsInstanceTypeChangedError(err error) bool {
 
 func IsClusterNotFoundError(err error) bool {
```

Comment:
This was previously part of this function in aws sdk v1, see previous commit https://github.com/aws/amazon-ecs-agent/blob/a19994a54b6b77221dc6cc00faea1e380d65520b/agent/vendor/github.com/aws/amazon-ecs-agent/ecs-agent/api/errors/errors.go#L65